insight-generator/sparkler_data.json

```diff
@@ -0,0 +1,24 @@
gion-footer-first\">\n    <div id=\"block-block-1\" class=\"block block-block\">\r\n\r\n<!-- begin nsidc_footer.ssi --><!-- vvvvvvvvvv SSI BELOW vvvvvvvvvv --><!-- nsidc_footer.ssi --><div class=\"clearGroup\" id=\"footerNSIDC\"><div class=\"footer-column grid_4\"><h2>Find Data<\/h2>&#13;\n&#13;\n\t\t<ul><li><a href=\"\/data\/data-search.html\">Search NSIDC Data<\/a><\/li>&#13;\n\t\t\t<li><a href=\"http:\/\/reverb.echo.nasa.gov\/reverb\/\">Reverb (NASA)<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/icebridge\/portal\/\">IceBridge Portal<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/data\/data_pool\/index.html\">Data Pool (Direct FTP for select data)<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/data\/collections.html\">Data Collections List<\/a><\/li>&#13;\n\t\t<\/ul><\/div>&#13;\n&#13;\n\t<div class=\"footer-column grid_4\"><h2>Stay Current<\/h2>&#13;\n&#13;\n\t\t<ul><li><a href=\"\/arcticseaicenews\/\">Arctic Sea Ice News and Analysis <\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/data\/seaice_index\/\">Sea Ice Index (Passive microwave satellite data)<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/data\/masie\/index.html\">MASIE (Daily sea ice extent, multi-source)<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/greenland-today\/\">Greenland Today<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/news\/newsroom\">Newsroom <\/a><\/li>&#13;\n\t\t<\/ul><\/div>&#13;\n&#13;\n\t<div class=\"footer-column grid_4\"><h2>Learn About Snow and Ice<\/h2>&#13;\n&#13;\n\t\t<ul><li><a href=\"\/icelights\/\">Icelights: Answers to Your Questions<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/cryosphere\/quick-facts\">Cryosphere Quick Facts <\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/cryosphere\/glaciers\/\">All About Glaciers<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/cryosphere\/snow\/\">All About Snow <\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/cryosphere\/seaice\/\">All About Sea Ice<\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/arcticmet\/\">Arctic Climatology - A Primer <\/a><\/li>&#13;\n\t\t<\/ul><\/div>&#13;\n&#13;\n\t<div class=\"footer-column grid_4\" id=\"footer-column-last\"><h2>Get Help<\/h2>&#13;\n&#13;\n\t\t<ul><li><a href=\"https:\/\/support.nsidc.org\/forums\">Knowledge Base<\/a><\/li>&#13;\n\t\t\t<li><a href=\"https:\/\/nsidc.zendesk.com\/account\/dropboxes\/20123172\" onclick=\"script: Zenbox.show(); return false;\">Ask Us<\/a><\/li>&#13;\n\t\t<\/ul><h2>Etc.<\/h2>&#13;\n&#13;\n\t\t<ul><li><a href=\"\/about\/use_copyright.html\">Use &amp; Copyright <\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/about\/privacy\">Web Policy <\/a><\/li>&#13;\n\t\t\t<li><a href=\"\/about\/jobs\/\">Jobs <\/a><\/li>&#13;\n\t\t<\/ul><div class=\"social-icon-container\"><a href=\"https:\/\/www.facebook.com\/NSIDC\"><img alt=\"Facebook\" src=\"\/images\/social-media\/facebook16x16.png\" \/><\/a> <a href=\"https:\/\/twitter.com\/NSIDC\"> <img alt=\"Twitter\" src=\"\/images\/social-media\/twitter16x16.png\" \/><\/a><\/div>&#13;\n\t<\/div>&#13;\n&#13;\n\t<div id=\"footer-bottom\"><div class=\"grid_9\"><p><img alt=\"NSIDC logo\" class=\"floatLeft\" height=\"34\" 
```

Comment:
do we need to check this in? This is specific sparkler data, right? we don't check this in?

Reply:
Actually, we need to modify Sparkler data to make it work with Insight-Generator, otherwise it will throw an error. I need to write a parser for it. So for now, I have added this file for testing purpose.